ev3dev/core.py

```diff
@@ -182,8 +184,16 @@ def get_index(file):
                 self._device_index = get_index(name)
                 self.connected = True
             except StopIteration:
+                self._path = None
+                self._device_index = None
                 self.connected = False
 
+    def __str__(self):
+        if 'address' in self.kwargs:
+            return "%s-%s" % (self.__class__.__name__, self.kwargs.get('address'))
```

Comment:
I think
```py
return "%s('%s')" % (self.__class__.__name__, self.kwargs.get('address'))
```
would look better (e.g. `MediumMotor('outA')`). It looks closer to what may actually be written in code, and hence more familiar.